pkgs/by-name/oa/oatmeal/package.nix

```diff
@@ -0,0 +1,63 @@
+{
+system ? builtins.currentSystem
+, pkgs
+, lib
+, fetchurl
+, installShellFiles
+}:
+let
+  shaMap = {
+    x86_64-linux = "0x4dn7g04ny2asw38059ida3044wps6cplgg63sf5pz9qz74i9v8";
+    aarch64-linux = "12i3agydmvqws01cki42kcms0jbf7pyhx8qkld6r5s8vs5ydg2nf";
+    x86_64-darwin = "1sijvqxkr0vpvqhfjlbdyllrzvv5c3ckf2fw2cfjzc2j1mamci4r";
+    aarch64-darwin = "1hncfa8598kv9wa3n090avvzd2pa67axq9ggvnz48ssp9fr4w7ar";
+  };
+
+  urlMap = {
+    x86_64-linux = "https://github.com/dustinblackman/oatmeal/releases/download/v0.12.1/oatmeal_0.12.1_linux_amd64.tar.gz";
+    aarch64-linux = "https://github.com/dustinblackman/oatmeal/releases/download/v0.12.1/oatmeal_0.12.1_linux_arm64.tar.gz";
+    x86_64-darwin = "https://github.com/dustinblackman/oatmeal/releases/download/v0.12.1/oatmeal_0.12.1_darwin_amd64.tar.gz";
+    aarch64-darwin = "https://github.com/dustinblackman/oatmeal/releases/download/v0.12.1/oatmeal_0.12.1_darwin_arm64.tar.gz";
+  };
+in
+pkgs.stdenv.mkDerivation {
```

Comment:
I see this is a Rust application. Why not use [buildRustPackage](https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/rust.section.md#buildrustpackage-compiling-rust-applications-with-cargo-compiling-rust-applications-with-cargo) instead?